app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -461,7 +461,9 @@ def final_order_logic(articles)
           articles.order(Arel.sql("RANDOM() ^ (1.0 / greatest(articles.comment_score, 0.1)) DESC"))
         when "final_order_by_random_weighted_to_last_comment_at"
           articles
-            .order(Arel.sql("RANDOM() ^ (1.0 / extract(epoch from now() - articles.last_comment_at)::integer) ASC"))
+            .order(
+              Arel.sql("RANDOM() ^ (1.0 / (1 + (extract(epoch from now() - articles.last_comment_at)::integer)) ASC"),
```

Comment:
there's an edge case, which I'll ignore, if the last comment at is set by a system where the clock is skewed from the db clock.

If this happens again we can use `MIN(1, extract - last)` but I assume that's not a problem.

Reply:
Hence the "always-ish" comment.  And I think we should go with a more robust solution